Building.md

```diff
@@ -94,7 +94,23 @@ mono main/build/bin/MonoDevelop.exe
 1. Launch, MonoDevelop
 2. Edit > Preferences > Projects > .NET Core Debuggers
 3. Click `...` and navigate to, `main/build/AddIns/Samsung.Netcoredbg/netcoredbg`
-4. Click, OK and start debugging
+4. Click, OK and start debugging  
+
+
+### Running DotDevelop with .NET6.0+ installed
+
+With dotnet-sdk-6.0+ installed, the following error occurs:-  
+
+"MSB4236 WorkloadAutoImportPropsLocator could not be found".  
+
+This is described by  [this issue](https://github.com/dotnet/sdk/issues/17461) with the following workaround:-   
+
+Set the environment variable `MSBuildEnableWorkloadResolver=false` prior to starting monodevelop  
+
+eg, in a terminal, before starting dotdevelop as above...   
+  
+    export MSBuildEnableWorkloadResolver=false

```

Comment:
The proper way to format command line should be using ``` blocks.

Reply:
@lextm  Thanks for picking that up.  For multi-line commands, I've tended to be to be a bit lax in differentiating between the 'triple-back-ticks'  method and the 'four leading spaces' approach, and didn't appreciate there was a difference.  I'll update the PR.